### PR TITLE
2843 longer urls

### LINKED ISSUE #2843: CKEditor: Long link URLs being truncated
## Issue
Editors using a basic subset of ckeditor tools are finding that when they paste long query style URL's into the modal box that is presented by the link function, that URL is truncated, which breaks the link.  Shorter links work as expected.

## Expected behavior
The link function accepts any valid link.

## Observed Workaround
Switching to HTML Editor allows the complete URL to be entered into the code and in view it works as expected.  Since exposing HTML Editor can be a security issue, it is preferred that the WYSIWYG editor work as expected.

##  Comments
Seems that input filtering controls this either in core or at the plugin level.  Would be nice if this was exposed as an admin setting so that it can be en/dis/abled or the value that controls the length of the truncate can be modified.

---

~PR by @alexfinnarn : https://github.com/backdrop/backdrop/pull/2032~ merged
PR by @jenlampton: https://github.com/backdrop/backdrop/pull/2940

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2843

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍

File: core/modules/views/plugins/views_plugin_display_page.inc [hunk 1]
```diff
@@ -264,7 +264,7 @@ class views_plugin_display_page extends views_plugin_display {
     $options['path'] = array(
       'category' => 'page',
       'title' => t('Path'),
-      'value' => views_ui_truncate($path, 24),
+      'value' => $path,
```
Comment (quicksketch) [inline core/modules/views/plugins/views_plugin_display_page.inc]: This change seems unrelated. And it's a display-only presentation, not a field. Should this be reverted?

**Comment by quicksketch** [comment]
Merged into 1.x and 1.14.x. Thanks!
